main.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"strconv"
+)
+
+func main() {
+
+	idStr := "1"
+	id, err := strconv.Atoi(idStr)
```

Comment:
**[unused]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
err declared but not used (compile)